API/projectClass/projectClass.m

```diff
@@ -157,13 +166,12 @@
                 error('experiment type has to be Standard Layers, Custom Layers or Custom XY');
             end
             
-            obj.ModelType = val;
+            obj.ModelType = lower(val);
```

Comment:
The legacyChanges function seems to be using a different project class, the type is accessed from a module object `problem.module.type` on line 3  and one of the types is 'Custom XY Profile' instead of 'custom xy', so I will leave it as is.